### PR TITLE
Reframed

### PR DESCRIPTION
**Pull request by lhtxa**
Updating reframed recipe
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by lhtxa** [comment]
@bioconda/core

**Review by rpetit3 — APPROVED** [review]

**Comment by rpetit3** [comment]
@BiocondaBot please fetch artifacts

File: recipes/reframed/meta.yaml [hunk 1]
```diff
@@ -43,7 +45,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - acaprez
-  identifiers:
-    - biotools:reframed
-    - doi:10.5281/zenodo.7955995
+    - lhtxa
+    - cdanielmachado
```
Comment (marcelm) [inline recipes/reframed/meta.yaml:49]: Is the person you add here fine with being added as a maintainer?

File: recipes/reframed/meta.yaml [hunk 2]
```diff
@@ -43,7 +45,5 @@ about:
 
 extra:
   recipe-maintainers:
-    - acaprez
```
Comment (marcelm) [inline recipes/reframed/meta.yaml:46]: I’m assuming you have a good reason for removing this user from the maintainer list, but just for completeness, can you state it?

Comment (lhtxa) [reply recipes/reframed/meta.yaml:46]: Hi! I am new here, so I am sorry if something wasn't done according to the good practices.
The new maintainer I added is the developer of reframed and is fine with being added as a maintainer here. 
I don't have a good reason to remove this user, I didn't contact him at all. Now that I think about it, I can see these might not be the appropriate way to go... so I will contact this user.

File: recipes/reframed/meta.yaml (hunk 1 above)
Comment (lhtxa) [reply recipes/reframed/meta.yaml:49]: Yes, the new maintainer I added is the developer of reframed and is fine with being added as a maintainer here.

File: recipes/reframed/meta.yaml [hunk 3]
```diff
@@ -43,7 +45,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - lhtxa
+    - cdanielmachado
     - acaprez
-  identifiers:
-    - biotools:reframed
-    - doi:10.5281/zenodo.7955995
```
Comment (bgruening) [inline recipes/reframed/meta.yaml:49]: why do you remove those?

File: recipes/reframed/meta.yaml (hunk 2 above)
Comment (acaprez) [reply recipes/reframed/meta.yaml:46]: Either way is fine with me.  If the developer of the package wants to directly be the recipe maintainer themselves, I have no objection and you can remove me from the list.

Comment (lhtxa) [reply recipes/reframed/meta.yaml:46]: I think it is fine. Thanks!

File: recipes/reframed/meta.yaml (hunk 3 above)
Comment (lhtxa) [reply recipes/reframed/meta.yaml:49]: Those are references where the package can be used but do not regard the package itself.

Comment (lhtxa) [reply recipes/reframed/meta.yaml]: They are back.

**Review by bgruening — APPROVED** [review]

**Pull request merged**